src/systems/userinput/paths.js

```diff
@@ -193,7 +197,8 @@ paths.device.daydream = {
   axis: axisName => {
     return `${daydream}axis/${axisName}`;
   },
-  pose: `${daydream}pose`
+  pose: `${daydream}pose`,
+  matrix: `${daydream}matrix`
```

Comment:
As we talked about in person, I think this should replace `pose` and not be called `matrix` but seems fine to do that as a fast follow since I agree that `pose` is used in some mysterious ways at the moment, so its more to debug